index.html

```diff
@@ -72,15 +71,18 @@
                 <li class="dropdown">
                     <a class="dropdown-toggle" data-toggle="dropdown" href="#" >Documentation <span class="caret"></span></a>
                     <ul class="dropdown-menu">
-                        <li><a href="https://cwiki.apache.org/confluence/display/PINOT" target="_blank">Apache Wiki <i class="fa fa-external-link"></i></a></li>
-                        <li class="divider"></li>
                         <li class="dropdown-header">ReadTheDocs</li>
                         <li><a href="http://pinot.readthedocs.io/en/latest/" target="_blank">Latest <i class="fa fa-external-link"></i></a></li>
+                        <li class="divider"></li>
+                        <li><a href="https://cwiki.apache.org/confluence/display/PINOT" target="_blank">Wiki <i class="fa fa-external-link"></i></a></li>
+                        <li><a href="https://cwiki.apache.org/confluence/display/PINOT/Blogs+and+Talks" target="_blank">Blogs and Talks <i class="fa fa-external-link"></i></a></li>
                     </ul>
                 </li>
                 <li class="dropdown">
                     <a class="dropdown-toggle" data-toggle="dropdown" href="#">Community <span class="caret"></span></a>
                     <ul class="dropdown-menu">
+                        <li><a href="https://cwiki.apache.org/confluence/display/PINOT/Get+Involved" target="_blank">Get Involved <i class="fa fa-external-link"></i></a></li>
```

Comment:
btw, I think the user group is `users@pinot.apache.org` not `user@pinot.apache.org`

Reply:
Good catch.

Reply:
Good Catch. Updated.

Reply:
We need to update the user group here too http://incubator.apache.org/projects/pinot.html. Maybe you or @kishoreg can help.